### ISSUE TITLE
unable to click to start on nintendo switch

### ISSUE THREAD

**Issue by Hoot679** [letsgoawaydev/gamesandstuff#11]

**Comment by letsgoawaydev** [comment]
yeah i have tried to fix this and i dont think its possible unless i use raw html, which i probably will do sometime in the distant future, i havent worked on this in a while

**Comment by Hoot679** [comment]
yeah i think porting something from haxe to html might take a while
